frontend/src/bundles/auth/auth-api.ts

```diff
@@ -67,6 +68,18 @@ class AuthApi extends HttpApi {
 
         return await response.json<UserWithProfileRelation>();
     }
+
+    public async updateAccessToken(): Promise<AuthTokenResponse> {
```

Comment:
```suggestion
    public async requestNewAccessToken(): Promise<AuthTokenResponse> {
```
this method is not updating the existing token, it's requesting a new one since the current token expired, maybe change the naming?